app/controllers/admin/petitions_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   end
 
   def threshold
-    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => 50)
```

Comment:
Would we gain anything by always using our custom search? It includes the pagination and the implementation would be uniform (`@petitions` would always be an instance of `Browseable:Search` everywhere - now it's mixed with `ActiveRecord:Relation`).